commands/id.go

```diff
@@ -37,7 +37,6 @@ var idCmd = &cmds.Command{
 		details := IDDetails{
 			Addresses: make([]ma.Multiaddr, len(addrs)),
 			ID:        hostID,
-			// TODO: fill out the other details
```

Comment:
Is there an issue corresponding to this?  I'm guessing that its being written but just flagging in case.